<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderCard</class>
 <widget class="QWidget" name="OrderCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <item>
    <widget class="QFrame" name="OrderStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #E8F0FE;
border-radius: 10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>8</number>
      </property>
      <item>
       <widget class="QPushButton" name="OrderDetailsBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	rotate: 0;
}

QPushButton:hover {
	rotate: 90deg;
	color: #eeeee;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/Icons/arrow_right.svg</normaloff>
          <normalon>:/icons/Icons/arrow_right.svg</normalon>
          <activeon>:/icons/Icons/arrow-down.svg</activeon>:/icons/Icons/arrow_right.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="OrderLabel">
        <property name="font">
         <font>
          <family>Franklin Gothic Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);</string>
        </property>
        <property name="text">
         <string>Order #PVQTKTDFWVI29-7702</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="OrderHSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="OrderStatusSelect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>175</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Franklin Gothic Medium</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LayoutDirection::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox {
    border-radius: 5px;
    padding: 9px 16px;
    color: white;
    background-color: green;
	width: 100%;
    text-align: center; /* Center-align the text */
}

QComboBox:hover {
    border-color: darkgreen;
    background-color: #32CD32; /* Lighter green for hover effect */
}

QComboBox::drop-down {
    width: 20px; /* Adjust width of the dropdown button */
    border: none;
}


QComboBox QAbstractItemView {
    background-color: white;
    color: black;
    selection-background-color: #32CD32; /* Highlight selected item in green */
    selection-color: black;
    padding: 5px;
    text-align: center;
    outline: none; /* Remove focus border for cleaner look */
}
</string>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Change Status</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>Being Prepared</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Out for Delivery</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Delivered</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteBtn">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="resource.qrc">
          <normaloff>:/icons/Icons/bin.svg</normaloff>:/icons/Icons/bin.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="OrderDetails" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #E8F0FE;
color: black;
border-radius: 10px;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_18">
      <property name="topMargin">
       <number>9</number>
      </property>
      <item>
       <widget class="QLabel" name="DetailsLabel">
        <property name="font">
         <font>
          <family>Franklin Gothic Medium</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Order Details:</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="DetailsList">
        <property name="topMargin">
         <number>4</number>
        </property>
        <property name="bottomMargin">
         <number>4</number>
        </property>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="OrderTypeLabel">
        <property name="font">
         <font>
          <family>Franklin Gothic Medium</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Order Type:</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="TotalPriceLabel">
        <property name="font">
         <font>
          <family>Franklin Gothic Medium</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Total Price: </string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextInteractionFlag::LinksAccessibleByMouse|Qt::TextInteractionFlag::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
